utils/envvars/parser.go

```diff
@@ -135,10 +145,25 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// parseBool validate bool as string value.
+func parseBool(value string) (bool, error) {
```

Comment:
Do we really need this function?

Reply:
This function can be reused for other boolean params in the futute and there is `parseStringDuration` func. But other boolean values parsed in-place, it's a bit strange. I can do the same, should i?

Reply:
removed